src/osdag/cad/common_logic.py

```diff
@@ -2050,6 +2054,12 @@ def display_3DModel(self, component, bgcolor):
             self.col = self.module_class()
             self.ColObj = self.createColumnInFrameCAD()
 
+            if self.component == "Model":
+                osdag_display_shape(self.display, self.ColObj, update=True)
+        elif self.mainmodule == 'Struts in Trusses':
+            self.S = self.module_class()
```

Comment:
Variable name too small and definitely error prone if someone else somewhere in the class decides to add a `self.S`. is `self.strut` not possible?

Reply:
This is a temporary placeholder, and will eventually be changed by Aryan or whoever will work on CAD for Strut Module.

Reply:
It's still error prone, and it seems there's not much use of the variable yet, so it'll be quick.

I do see a `self.s` being used at other places in the code. If someone else tries to make a temporary measure elsewhere and define `S` in the file, his could overwrite that.

Reply:
OK, so do you want me to change that? I will make sure to avoid using smaller variables in the future. Also you are seeing self.S is being used in other places in the common_logic.py?

Reply:
> OK, so do you want me to change that? ... Also you are seeing self.S is being used in other places in the common_logic.py?

Since I need you to rebase and resolve a regression, why not do that as well :)? And no, not used in `common_logic.py`, but that's a poor excuse.

> I will make sure to avoid using smaller variables in the future.

Thank you.